include/tcpdf_cmssignature.php

```diff
@@ -0,0 +1,545 @@
+<?php
```

Comment:
If this file could have a code formatting that would be great.
Else, it looks okay to me

Reply:
code formatting.. like what?
Sorry, I do not understand what you mean. :)